media/src/androidMain/kotlin/dev/icerock/moko/media/picker/ImagePickerFragment.kt

```diff
@@ -89,6 +89,7 @@ class ImagePickerFragment : Fragment() {
         )
     }
 
+    @Suppress("UnreachableCode")
```

Comment:
why itis unreachable?

Reply:
I don't know why, but detect is sure that all code inside onActivityResult is unreachable.
I checked if it actually works, and it works
So I add a Suppress annotation